### PR TITLE
feat: migrate tiflow verifyci to ksyun

### PR DESCRIPTION
**Pull request by purelind**
migration tiflow verifyci from idc to ksyun

### CODE REVIEW

File: jobs/pingcap/tiflow/latest/ghpr_verify.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        // priority(0) // 0 fast than 1
+        githubProjectUrl("https://github.com/pingcap/tiflow")
+        pipelineTriggers {
+            triggers {
+                ghprbTrigger {
+                    cron('H/5 * * * *')
+                    gitHubAuthId('') // using the default only one.
+
+                    triggerPhrase('.*/(run-verify.*)|(/merge)')
+                    onlyTriggerPhrase(false)
```
Comment (wuhuizuo) [inline jobs/pingcap/tiflow/latest/ghpr_verify.groovy:21]: it's will be triggered automatic when PR refreshed or created, I think it's no need to recall it when user comment with '/merge'

Comment (wuhuizuo) [reply jobs/pingcap/tiflow/latest/ghpr_verify.groovy:21]: Please check rest parts.

File: pipelines/pingcap/tiflow/latest/ghpr_verify.groovy [hunk 2]
```diff
@@ -0,0 +1,111 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master and latest release branches
+// @Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tidb"
```
Comment (wuhuizuo) [inline pipelines/pingcap/tiflow/latest/ghpr_verify.groovy:6]: should using `jenkins-tiflash` ?

File: pipelines/pingcap/tiflow/latest/ghpr_verify.groovy [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            yamlFile POD_TEMPLATE_FILE
+            defaultContainer 'golang'
+        }
+    }
+    environment {
+        FILE_SERVER_URL = 'http://fileserver.pingcap.net'
+    }
+    options {
+        timeout(time: 40, unit: 'MINUTES')
+        // parallelsAlwaysFailFast()
```
Comment (wuhuizuo) [inline pipelines/pingcap/tiflow/latest/ghpr_verify.groovy:24]: should we make it fail fast?

File: staging/jobs/pingcap/tiflow/latest/ghpr_verify.groovy [hunk 4]
```diff
@@ -17,7 +17,7 @@ pipelineJob('pingcap/tiflow/ghpr_verify') {
                     cron('H/5 * * * *')
                     gitHubAuthId('') // using the default only one.
 
-                    triggerPhrase('.*/(run-(debug-verify).*)')
+                    triggerPhrase('.*/((run-debug-verify|verify).*)')
```
Comment (wuhuizuo) [inline staging/jobs/pingcap/tiflow/latest/ghpr_verify.groovy:20]: when prod job ready, we should disable the staging job, rest same suggestion.

File: jobs/pingcap/tiflow/latest/ghpr_verify.groovy (hunk 1 above)
Comment (purelind) [reply jobs/pingcap/tiflow/latest/ghpr_verify.groovy:21]: This is a current user habit, and this behavior change requires advance notice. This time only migration will be performed

File: pipelines/pingcap/tiflow/latest/ghpr_verify.groovy (hunk 3 above)
Comment (purelind) [reply pipelines/pingcap/tiflow/latest/ghpr_verify.groovy:24]: ci will execute multiple ut tasks in parallel, so  I don't recommend to make ti fail fast

File: pipelines/pingcap/tiflow/latest/ghpr_verify.groovy (hunk 2 above)
Comment (purelind) [reply pipelines/pingcap/tiflow/latest/ghpr_verify.groovy:6]: just use the same namespace

File: staging/jobs/pingcap/tiflow/latest/ghpr_verify.groovy (hunk 4 above)
Comment (purelind) [reply staging/jobs/pingcap/tiflow/latest/ghpr_verify.groovy:20]: yes, i will diable those staging job after prod is ready to work

File: pipelines/pingcap/tiflow/latest/ghpr_verify.groovy (hunk 2 above)
Comment (purelind) [reply pipelines/pingcap/tiflow/latest/ghpr_verify.groovy]: use `jenkins-tiflow`

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**